internal/app/connectconformance/test_case_library.go

```diff
@@ -85,6 +85,21 @@ func newTestCaseLibrary(
 }
 
 func (lib *testCaseLibrary) expandSuite(suite *conformancev1alpha1.TestSuite, configCases map[configCase]struct{}) error {
+	if suite.ReliesOnTlsClientCerts && !suite.ReliesOnTls {
+		return fmt.Errorf("suite %q is misconfigured: it relies on TLS client certs but not TLS", suite.Name)
+	}
+	if suite.ReliesOnConnectGet && !only(suite.RelevantProtocols, conformancev1alpha1.Protocol_PROTOCOL_CONNECT) {
+		return fmt.Errorf("suite %q is misconfigured: it relies on Connect GET support, but has unexpected relevant protocols: %v", suite.Name, suite.RelevantProtocols)
+	}
+	if suite.ConnectVersionMode == conformancev1alpha1.TestSuite_CONNECT_VERSION_MODE_IGNORE && !only(suite.RelevantProtocols, conformancev1alpha1.Protocol_PROTOCOL_CONNECT) {
+		return fmt.Errorf("suite %q is misconfigured: it relies on Connect GET support, but has unexpected relevant protocols: %v", suite.Name, suite.RelevantProtocols)
```

Comment:
Is this the right error message? I think we'd want something here about ignoring Connect version mode, right?

Reply:
Yep, thank you for catching that. Copy+pasta error on my part.